schema/2.0.0/corpora_schema.md

```diff
@@ -145,13 +145,13 @@ The following ontology dependencies are *pinned* for this version of the schema.
 
 cellxgene requires ENSEMBL gene identifiers to ensure that all datasets it stores measure the same features and can therefore be integrated.
 
-The following gene annotation dependencies are *pinned* for this version of the schema.
+The following gene annotation dependencies are *pinned* for this version of the schema. For multi-organism experiments, cells from any organism are allowed as long as orthologs from the following organism annotations are used.
 
 | Organism | Required version | Download |
```

Comment:
Has there been any consideration for spike-ins? We have come across a couple existing datasets that includes spike-ins.

Reply:
I had not thought about this and we should definitely consider it, we can either be strict and allow certain spike-ins although I'm afraid generalization may not be possible, or allow for a gene prefix (e.g. "SPIKE-IN-[...]")